pkg/image/qemu.go

```diff
@@ -192,3 +193,19 @@ func reportProgress(line string) {
 		progress.WithLabelValues(ownerUID).Set(v)
 	}
 }
+
+// CreateBlankImage creates empty raw image
+func CreateBlankImage(dest, size string) error {
+	glog.V(1).Infof("creating raw image with size %s", size)
+	return qemuIterface.CreateBlankImage(dest, size)
+}
+
+// CreateBlankImage creates a raw image with a given size
+func (o *qemuOperations) CreateBlankImage(dest, size string) error {
```

Comment:
Please rebase and look at the Resize function and follow that pattern. The size is a Quantity there and there is a function that translates the Quantity into a qemu compatible string. Right now this code won't work for a value in the K range.

Reply:
done